### PR TITLE
493 breaking news

### PR DESCRIPTION
**Pull request by JamieRL**
- Added get breaking news helper to article helper that checks to see if there are any published, breaking articles that are less than two hours old
- Added a breaking news banner to the default frontpage widget

### CODE REVIEW

**Comment by JamieRL** [comment]
I only added this to the default frontpage widget. Should I add it to the other frontpage widget templates?

**Comment by JamieRL** [comment]
Nvm I added it to the hompage base template

File: ubyssey/static/src/styles/modules/_homepage.scss [hunk 1]
```diff
@@ -553,6 +553,27 @@ ol.article-list {
   }
 }
 
+.o-breaking-news{
+  width: 100%;
+  display:flex;
+  border-radius: 3px;
+  background-color:#e41818;
+
+
```
Comment (Rowansdabomb) [inline ubyssey/static/src/styles/modules/_homepage.scss]: extra whitespace

File: ubyssey/static/src/styles/modules/article/_base.scss [hunk 2]
```diff
@@ -29,6 +29,29 @@ div.indicator {
...
+  h3 {
+    padding-left: 1rem;
+    color: white;
+    p {
+      color: white;
+      font-weight: lighter;
+    }
+  }
+
+  // @media($bp-larger-than-tablet) {
```
Comment (Rowansdabomb) [inline ubyssey/static/src/styles/modules/article/_base.scss]: remove old comments

**Review by Rowansdabomb — DISMISSED** [review]
looks good, just a couple comments

**Pull request merged**